in_toto/matchrule_validators.py

```diff
@@ -37,46 +38,51 @@ def _validate_match_rule(keywords):
         step) = keywords
   else:
     raise securesystemslib.exceptions.FormatError("Wrong rule format,"
-      " should be: MATCH (MATERIAL/PRODUCT)"
+      " should be: MATCH (MATERIAL | PRODUCT)"
       " <path_pattern> [AS <target_path_pattern>] FROM <step>.\n\t"
-      "Got: {}".format(" ".join(keywords)))
+      "Got: {}".format(keywords))
 
   if rule != "MATCH" and rule.upper() != "MATCH":
-    raise securesystemslib.exceptions.FormatError(
-        "Wrong rule to verify! {}".format(rule))
+    raise securesystemslib.exceptions.FormatError("Wrong rule format.\n\t"
+        "Got: {}".format(keywords))
```

Comment:
The user could probably figure it out on their own, but a more helpful exception would emphasize that the rule keyword is wrong (i.e., must be `MATCH` or `match`).